docs/uberon-release.md

```diff
@@ -0,0 +1,63 @@
+# Uberon Release workflow
+
+While Uberon is an ODK ontology, it has a specific workflow for releases due to its large size and the limitations on standard GitHub releases.
+
+## Requirements
+
+Aside from the standard requirements needed for ODK workflow, `GH` is required. 
+Instructions on how to install `GH` can be found [here](https://cli.github.com/manual/installation)
+
+You will need to log in to your GitHub account on `GH` before you can uses it. To do this, enter the following in your terminal:
+```
+gh auth login
+```
+
+You can then follow instructions below for web browser login (or use your prefer means of logging in).
+```
+% gh auth login
+? What account do you want to log into? GitHub.com
+? What is your preferred protocol for Git operations? SSH
+? Generate a new SSH key to add to your GitHub account? Yes
+? Enter a passphrase for your new SSH key (Optional) 
+? Title for your SSH key: GitHub SSH
+? How would you like to authenticate GitHub CLI? Login with a web browser
+
+! First copy your one-time code: XXXX-XXXX
+Press Enter to open github.com in your browser... 
+✓ Authentication complete.
+- gh config set -h github.com git_protocol ssh
+✓ Configured git protocol
+✓ Uploaded the SSH key to your GitHub account: /Users/username/.ssh/id_ed25519.pub
+✓ Logged in as username
+```
+
+## Release Process
+
+1. Follow the preparation steps found in the [ODK-workflow release document](odk-workflows/ReleaseWorkflow.md)
```

Comment:
probably needs fixing now - add everything here.

Reply:
I think for preparation it makes sense to copy the whole thing over, but for review theres a clear section so redirect would suffice